### PR TITLE
[14.0][mig]autovacuum_message_attachment

### PR DESCRIPTION
**Pull request by hkapatel-initos**

### CODE REVIEW

**Review by fshah-initos — APPROVED** [review]
LGTM. Code review

**Review by rconjour — APPROVED** [review]
LGTM

**Comment by hkapatel-initos** [comment]
@florian-dacosta can you please review this pr?

**Comment by hkapatel-initos** [comment]
@etobella it would be nice if you review it here.

**Comment by hkapatel-initos** [comment]
Hello maintainers,
It would be nice if you are trying to merge it

**Comment by hkapatel-initos** [comment]
Hello maintainers,
It would be nice if you are trying to merge it

File: autovacuum_message_attachment/models/autovacuum_mixin.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Copyright (C) 2019 Akretion
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
-import datetime
 import logging
 
 import odoo
 from odoo import api, models
-from odoo.tools.safe_eval import safe_eval
+from odoo.tools.safe_eval import datetime, safe_eval
```
Comment (gurneyalex) [inline autovacuum_message_attachment/models/autovacuum_mixin.py:8]: why did you change the import path of `datetime`?

Comment (hkapatel-initos) [reply autovacuum_message_attachment/models/autovacuum_mixin.py:8]: Resolve this error ;
Error: Module <module 'datetime' from '/usr/lib/python3.6/datetime.py'> can not be used in evaluation contexts

Prefer providing only the items necessary for your intended use.

If a "module" is necessary for backwards compatibility, use
`odoo.tools.safe_eval.wrap_module` to generate a wrapper recursively
whitelisting allowed attributes.

And according to odoo 14 import path add like this.

Comment (gurneyalex) [reply autovacuum_message_attachment/models/autovacuum_mixin.py:8]: thx

**Review by gurneyalex — APPROVED** [review]
